### ISSUE TITLE
Create a POC page which shows the search scores along side the article

### ISSUE THREAD

**Issue by roman87** [TeamMentor/TM_4_0_Design#517]

**Comment by luchia** [comment]
I'll take this and create the template for you, @DinisCruz, so all you need to do is put the logic in.

**Comment by luchia** [comment]
Hey @DinisCruz, I created the template but I can't see it in the browser. I think it's something to do with the data again -

![screen shot 2015-03-11 at 8 43 16 am](https://cloud.githubusercontent.com/assets/4547737/6587017/bc27fdf0-c7ca-11e4-8271-288187710fdc.png)

This is my error log -

![screen shot 2015-03-11 at 8 43 56 am](https://cloud.githubusercontent.com/assets/4547737/6587025/c812cc12-c7ca-11e4-92ad-dd30d0742da7.png)

**Comment by DinisCruz** [comment]
fixed on latest dev (I had to move and rename the jade file, so that it is in a location where it will be 'feed' the query data)

this page is now on **http://localhost:1337/poc/filters-results** and looks like this

![](https://cloud.githubusercontent.com/assets/656739/6722431/48619d88-cdd1-11e4-8b6d-d438f30bf5bf.png)

See https://github.com/TeamMentor/TM_4_0_Design/issues/602 for more details

**Comment by DinisCruz** [comment]
@luchia btw, the error you were having (img bellow) was caused by mixing tabs and spaces

https://cloud.githubusercontent.com/assets/4547737/6587025/c812cc12-c7ca-11e4-92ad-dd30d0742da7.png

**Comment by luchia** [comment]
Hey @roman87, this is done and can be seen on the POC page (filters-result).

**Comment by roman87** [comment]
Hmm. ok but the page does not actually show search results as this issue describes :)

**Comment by DinisCruz** [comment]
you mean the search scores? humm, how is this supposed to work?

Is this just for PoC or will the users see this on both index and search views?

**Comment by roman87** [comment]
Well yes, this was the point of the issue :) This is just for the POC, so we can see what the search score was for the article. We can just fit the score and the breakdown below the title. It doesn't need to be aesthetic, just functional.
Makes sense?

**Comment by DinisCruz** [comment]
The prob is that at the moment the PoC is not really 'navigatable' 

I.e. when you click on a link on either left or right queries/filters you will be back to the current live UI.

So do we need this here?

I think a more interresting place to show this data would be on the search results

**Comment by roman87** [comment]
well yes. yes. that is what I am asking :). POC page so you do a search, and then the search results show the scores. Makes sense?

**Comment by DinisCruz** [comment]
yeah, that will need a more complex PoC backend that we currently have (not that hard, but not in the 'out of the box' PoC model we have now)

I've started looking at it since more and more this will be very useful

We also need to fix the PoC pages in the beta.teammentor.net

**Comment by roman87** [comment]
Ok cool. remember this is sprint 3, so not really a priority now.

**Comment by DinisCruz** [comment]
I think it is better if this is done as part of the Flare implementation

**Comment by roman87** [comment]
Agreed, sound good.

**Comment by roman87** [comment]
Changed the title a bit to reflect it.
